A3A/addons/core/functions/Supports/fn_SUP_UAV.sqf

```diff
@@ -29,9 +29,12 @@ private _planeType = selectRandom (Faction(_side) get "uavsAttack");
 private _aggro = if(_side == Occupants) then {aggressionOccupants} else {aggressionInvaders};
 if (_delay < 0) then { _delay = (0.5 + random 1) * (300 - 15*tierWar - 1*_aggro) };
 
-// Well, it doesn't actually hit anything...
-// ["_side", "_supptype", "_basetype", "_target", "_endtime", "_duration", "_power"]
-//A3A_supportStrikes pushBack [_side, "UAV", "TARGET", _target, time + 1200, 1200, 200];
+// Well, it does actually hit
+private _targArray = [];
+if (_target isEqualType objNull and {!isNull _target}) then {
+    A3A_supportStrikes pushBack [_side, "TARGET", _target, time + 1200, 1200, 200];
+    _targArray = [_target, _targPos];
+};
```

Comment:
Does this make it more accurate or prevent a strike on the same place?

Reply:
no idea about that one, just saw that all other supports use it, soo...